data.json

```diff
@@ -47,6 +48,11 @@
       "normal":38961, 
       "young":29616, 
       "research":0
+    },
+    "2022":{
```

Comment:
Is this the "30% rule"? For 2022 these numbers are increased. Normal: 39647, Young: 30001 (as published at https://zoek.officielebekendmakingen.nl/stcrt-2021-48308.html in Article VII.B

Reply:
thx for noticing!
fixed